docs/diagrams/ModelClassDiagram.puml

```diff
@@ -40,11 +43,18 @@ Person *--> Name
 Person *--> Phone
 Person *--> Email
 Person *--> Address
+Person *--> Class
+Person *--> Money
+Person *--> AdditionalNotes
```

Comment:
Should it be Class, Money, Phone or, Class, MoneyOwed, MoneyPaid, Phone, NokPhone? Might need to discuss with the rest of the team
@sjoann @cadencjk @Echomo-Xinyu @gnahzeus

Reply:
I think here it should display all the fields of `Person` and therefore in my pr, I have included all the fields.